### DISCUSSION TITLE
Force shallow on back button?

### DISCUSSION THREAD

**Discussion by greynguyen** [vercel/next.js#22817] [Help]
I am trying to not refresh and re-render data when the user clicks on the back button in the browser. I know this can be done with `Link` and `shallow` option.

I was just wondering is there a way that we can force the browser back button to be `shallow` as well? (I mainly want it to be shallow so that it can retain the scroll position on clicking back) Thank you!

**Comment by shuding** [comment]
If you want to retain the scroll position on navigations, you can try using `scroll={false}` to `<Link>`. Related documentation: https://nextjs.org/docs/api-reference/next/link#disable-scrolling-to-the-top-of-the-page.

**Reply by greynguyen** [reply]
Thank you for the response. But as I mentioned above. I am trying to make it shallow without using the Link component. This is trying to use the back button of the web browser
